configure.ac

```diff
@@ -1955,6 +1955,13 @@ if test "x$mono_feature_disable_gac" = "xyes"; then
 	AC_MSG_NOTICE([Disabled GAC support])
 fi
 
+# Currently, we only want to use DllMap for non-netcore builds
+if test "x$with_core" = "xno" -a "x$mono_feature_disable_dllmap" != "xyes"; then
+    AC_DEFINE(ENABLE_DLLMAP,1,[Enables use of DllMaps in MonoVM])
```

Comment:
Shouldn't this be a DISABLE_DLLMAP like the other defines ?

Reply:
I could make it one? In this case I originally viewed it more as a feature that's potentially turned on since I didn't have it as part of enable-minimal, but it doesn't matter to me - happy to change if you want.

Reply:
Its fine either fine, DISABLE_ is better for consistency.

Reply:
Renamed.